### PR TITLE
ship to prod

### PR DESCRIPTION
**Pull request by PierreJeanjacquot**

### CODE REVIEW

**Comment by PierreJeanjacquot** [comment]
preview: https://documentations.iex.ec/staging

File: help/migration.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  * [Application migration](migration#application-migration)
+  * [Dataset migration](migration#dataset-migration)
+  * [Requester migration](migration#requester-migration)
+* [JS library](migration#js-library)
+
+## CLI users
+
+### Application migration
+
+Application developers needs to publish their application sell order to the v7 marketplace in order to make it availlable for requesters.
```
Comment (james-toussaint) [inline help/migration.md]:
```suggestion
Application developers needs to publish their application sell order to the v7 marketplace in order to make it available for requesters.
```

File: help/migration.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+Publish your application to the v7 marketplace:
+
+```bash
+iexec app publish <app-address> [options]
+```
+
+### Dataset migration
+
+Dataset developers needs to push their dataset secret to the v7 SMS and publish their dataset sell order to the v7 marketplace in order to make it availlable for requesters.
```
Comment (james-toussaint) [inline help/migration.md]:
```suggestion
Dataset developers needs to push their dataset secret to the v7 SMS and publish their dataset sell order to the v7 marketplace in order to make it available for requesters.
```

**Review by james-toussaint — APPROVED** [review]

File: help/migration.md (hunk 1 above)
Comment (PierreJeanjacquot) [reply help/migration.md]: #90

File: help/migration.md (hunk 2 above)
Comment (PierreJeanjacquot) [reply help/migration.md]: #90

File: for-developers/v6-to-v7-migration.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+  * [Application migration](v6-to-v7-migration#application-migration)
+  * [Dataset migration](v6-to-v7-migration#dataset-migration)
+  * [Requester migration](v6-to-v7-migration#requester-migration)
+* [JS library](v6-to-v7-migration#js-library)
+
+## CLI users
+
+### Application migration
+
+Application developers needs to publish their application sell order to the v7 marketplace in order to makes it availlable for requesters.
```
Comment (ericr6) [inline for-developers/v6-to-v7-migration.md]: needs -> need
availlable -> available

**Review by ericr6 — CHANGES_REQUESTED** [review]

**Review by luisccastillo — APPROVED** [review]

**Review by ericr6 — APPROVED** [review]

File: for-developers/v6-to-v7-migration.md (hunk 3 above)
Comment (PierreJeanjacquot) [reply for-developers/v6-to-v7-migration.md]: #91

**Pull request merged**